boa/src/builtins/array/mod.rs

```diff
@@ -276,6 +276,29 @@ impl Array {
         Ok(array_obj_ptr)
     }
 
+    /// Utility function for concatenating array objects.
+    ///
+    /// Returns a Boolean valued property that if `true` indicates that
+    /// an object should be flattened to its array elements
+    /// by `Array.prototype.concat`.
+    pub(crate) fn is_concat_spreadable(this: &Value, context: &mut Context) -> Result<bool> {
+        if !this.is_object() {
+            return Ok(false);
+        }
+        let spreadable = this
+            .get_field(WellKnownSymbols::is_concat_spreadable(), context)
+            .unwrap_or_else(|_| Value::undefined());
```

Comment:
```suggestion
            .get_field(WellKnownSymbols::is_concat_spreadable(), context)?;
```
It should propagate the exception according to spec